packages/context/package.json

```diff
@@ -6,25 +6,30 @@
     "node": ">=6"
   },
   "scripts": {
-    "acceptance": "node ../../bin/select-dist mocha --opts ../../test/mocha.opts 'DIST/test/acceptance/**/*.js'",
+    "acceptance":
+      "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/acceptance/**/*.js'",
     "build": "npm run build:dist && npm run build:dist6",
-    "build:current": "node ../../bin/compile-package",
-    "build:dist": "node ../../bin/compile-package es2017",
-    "build:dist6": "node ../../bin/compile-package es2015",
-    "build:apidocs": "node ../../bin/generate-apidocs",
+    "build:current": "lb-tsc",
+    "build:dist": "lb-tsc es2017",
+    "build:dist6": "lb-tsc es2015",
+    "build:apidocs": "lb-apidocs",
     "clean": "rm -rf loopback-context*.tgz dist* package",
     "prepublish": "npm run build && npm run build:apidocs",
     "pretest": "npm run build:current",
-    "test": "node ../../bin/select-dist mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js' 'DIST/test/acceptance/**/*.js'",
-    "unit": "node ../../bin/select-dist mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js'",
-    "verify": "npm pack && tar xf loopback-context*.tgz && tree package && npm run clean"
+    "test":
+      "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js' 'DIST/test/acceptance/**/*.js'",
+    "unit":
+      "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js'",
+    "verify":
+      "npm pack && tar xf loopback-context*.tgz && tree package && npm run clean"
```

Comment:
Same comment applies here - please revert whitespace changes, `npm` is going to revert them anyways.

Please do the same in all `package.json` files modified by this PR.